### PR TITLE
Affiliate features

### PR DESCRIPTION
**Pull request by rkashapov**

### CODE REVIEW

**Comment by rkashapov** [comment]
This PR is based on https://github.com/orgnc/wordpress-plugin/pull/60
Review only last commit, please

**Review by g1r0 — APPROVED** [review]
LGTM

**Review by Mwindo — APPROVED** [review]
LGTM. Thanks, @rkashapov!

File: affiliate/src/ProductSearchModal.jsx [hunk 1]
```diff
@@ -1,63 +1,34 @@
...
   }, [onProductSelect]);
   return (
     <Modal
+      className="product-search-modal"
       onRequestClose={onClose}
       shouldCloseOnClickOutside={false}
       title="Product Search"
       isFullScreen
     >
       {/* TODO rkashapov: change to product-search page */ }
```
Comment (g1r0) [inline affiliate/src/ProductSearchModal.jsx:26]: Is this TODO obsolete?

File: affiliate/src/productCard/Edit.jsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ const Edit = ({ attributes, setAttributes }) => {
     [setAttributes],
   );
   const setDisplayDescription = useCallback(
-    (displayDescription) => setAttributes({ displayDescription }),
+    (displayDescripton) => setAttributes({ displayDescripton }),
```
Comment (g1r0) [inline affiliate/src/productCard/Edit.jsx:90]: mispell

**Review by g1r0 — APPROVED** [review]
LGTM

**Review by sergunich — APPROVED** [review]
👍

**Pull request merged**